src/utils/GitLabUtils/GitLabUtils.js

```diff
@@ -14,13 +14,17 @@ const getDailyContributions = (gitLabCalendar, date) => {
 
 export const mergeCalendarsContributions = (
   actualCalendar,
-  gitLabUserJsonCalendar
+  gitLabUserJsonCalendar,
+  from
 ) => {
   const copiedActualCalendar = javaScriptUtils.deepCopyObject(actualCalendar);
 
   copiedActualCalendar.children[0].children.forEach((weeklyData, weekIndex) => {
-    weeklyData.children.forEach((dailyData, dayIndex) => {
-      if (dailyData.attributes["data-count"]) {
+    weeklyData.children
+      .filter(dailyData =>
+        calendarUtils.filterContributionDays(dailyData, from)
```

Comment:
Should there be filter for `endDate` as well?

Reply:
https://github.com/c-hive/team-contribution-calendar/pull/64#discussion_r399778117

Doesn't this mean we don't support end dates?